nestor_api/lib/docker.py

```diff
@@ -0,0 +1,77 @@
+"""Docker library"""
+
+import nestor_api.lib.config as config
+import nestor_api.lib.git as git
+import nestor_api.lib.io as io
+from nestor_api.utils.logger import Logger
+
+
+def build(app_name, context):
+    """Build the docker image of the last version of the app"""
+    repository = context.get("repository")
+    image_tag = git.get_last_tag(repository)
+
+    Logger.info(
+        {"tag": f"${app_name}@${image_tag}"}, "Application name and version",
+    )
+
+    if has_docker_image(app_name, image_tag):
+        Logger.info({}, "Docker image already built (skipped)")
+        return image_tag
+
+    app_config = config.get_app_config(app_name)
+    commit_hash = context.get("commit_hash")
+    build_variables = app_config.get("build").get("variables")
+
+    # Application build environment variables:
+    builds_args = []
+    for key, value in build_variables.items():
+        builds_args.append(f"--build-arg {key}={value}")
+
+    builds_args.append(f"--build-arg COMMIT_HASH={commit_hash}")
+    builds_args_str = ' '.join(builds_args)
+
+    command = f'docker build --tag {app_name}:{image_tag} {builds_args_str} {repository}'
+
+    Logger.debug({command}, "Docker build command")
```

Comment:
Be careful, this creates a `set` and not a `dict` (js again 🙂 )
```suggestion
    Logger.debug({'command': command}, "Docker build command")
```